task1.js

```diff
@@ -0,0 +1,8 @@
+// Write a function that takes any value and returns its type as a string.
+
```

Comment:
in general it OK. But exist few cases which that  solution will not cover
`getType(null)` will return `object` but it should be `null`
`Arrays`, `functions`, and `other specific types` like Date, RegExp should be identified properly.